sdk/identity/identity/tsconfig.json

```diff
@@ -9,6 +9,6 @@
       "@azure/identity": ["./src/index"]
     }
   },
-  "include": ["src/**/*", "test/**/*", "samples-dev/**/*.ts"],
+  "include": ["src/**/*", "test/**/*", "samples-dev/**/*.ts", "../identity-common/src/util/logging.ts", "../identity-common/src/msal/nodeFlows/msalNodeCommon.ts", "../identity-common/src/util/tenantIdUtils.ts", "../identity-common/src/msal/flows.ts", "../identity-common/src/msal/utils.ts", "../identity-common/src/errors.ts", "../identity-common/src/client/identityClient.ts", "../identity-common/src/tokenCredentialOptions.ts", "../identity-common/src/regionalAuthority.ts", "../identity-common/src/msal/nodeFlows/tokenCachePersistenceOptions.ts", "../identity-common/src/constants.ts", "../identity-common/test/msalTestUtils.ts", "../identity-common/src/util/processMultiTenantRequest.ts", "../identity-common/src/credentials/multiTenantTokenCredentialOptions.ts", "../identity-common/src/util/processMultiTenantRequest.browser.ts", "../identity-common/src/msal/credentials.ts", "../identity-common/src/msal/types.ts", "../identity-common/src/util/identityTokenEndpoint.ts", "../identity-common/src/util/tracing.ts", "../identity-common/src/credentials/managedIdentityCredential/utils.ts"],
```

Comment:
why do we need to include `identity-common` files here? I thought each of these would be imported from the dependency instead of compiled by TS?